framework/platform/ios/tcuIOSPlatform.mm

```diff
@@ -28,6 +28,7 @@
 #include "glwInitES20Direct.hpp"
 #include "glwInitES30Direct.hpp"
 
+#include <OpenGLES/ES3/gl.h>
```

Comment:
Why do you need this include here?

It should be sufficient to have it in
framework/egl/egluStaticESLibrary.cpp and
framework/opengl/wrapper/glwInitES30Direct.cpp

Reply:
Without this I get:
```
VK-GL-CTS/framework/platform/ios/tcuIOSPlatform.mm:241:45: error: use of undeclared identifier 'GL_RENDERBUFFER'
        if (![getEAGLContext() renderbufferStorage:GL_RENDERBUFFER fromDrawable:(CAEAGLLayer *)m_layer])
```

Is there a better place I should put it? Maybe in the tcuIOSPlatform.hh next to the existing `#import <OpenGLES/EAGL.h>` include?